### PR TITLE
Add Support for Config

### PR DESCRIPTION
**Pull request by Ririio**
I added an "input" folder to ensure that we can test files in one singular directory as to having to add them manually each time. I also have the "ssg-config.json" located at the bottom of the file for when the user decides to run a script.

Updated the README to add the "config" in the table.

I can readjust if you happen to have any problems with the code

### CODE REVIEW

File: src/main.js [hunk 1]
```diff
@@ -12,6 +12,8 @@ module.exports.main = (args) => {
     dispatch("-h");
   } else if (args[0] === "-i" || args[0] === "--input") {
     dispatch("-i", args);
+  } else if (args[0] === "-c" || args[0] === "--input") {
```
Comment (SerpentBytes) [inline src/main.js:15]: There's a typo here.

Comment (Ririio) [reply src/main.js:15]: I thought I fixed this already weird

Comment (Ririio) [reply src/main.js]: It should be fixed now

**Review by SerpentBytes — CHANGES_REQUESTED** [review]
replace "--input" with "--config"

**Review by Boldgainobu — APPROVED** [review]
https://github.com/SerpentBytes/ririio-ssg

**Review by Boldgainobu — COMMENTED** [review]
https://github.com/Ririio/siteit/tree/issue-20

**Comment by SerpentBytes** [comment]
1. Using config flag and passing a file name that ends with an invalid extension prints testing `console.log`  and multiple error messages.

I entered the following: 
```
siteit --config ../input.md
```

Terminal output: 
```
-- Invalid or multiple source supplied --
-c
-- ERROR --
siteit -c myfile.json
usage: siteit --config myfile.json
```
2. Passing a valid file does not generate status and success messages like functions responsible for parsing `.txt` and `.json` files.
3. Passing an input file with the following `.json` config file renders inaccurate HTML. See below:

```
{
  "input" : "./inputs"
}
```

Output in terminal :

```
-- Attempting to create output directory --
-- Output directory exists. Attempting to delete it --
-- Pre-existing output directory deleted --
-- DONE creating output directory: /dist --
-- input.html generated in dist directory --

```

HTML rendered:

```
<!DOCTYPE html>
<html lang="en">
  <head>
    <meta charset="utf-8">
    <link rel="stylesheet" type="text/css" href="../src/siteit.css" />
    <link rel="preconnect" href="https://fonts.googleapis.com">
    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
    <link href="https://fonts.googleapis.com/css2?family=Questrial&display=swap" rel="stylesheet">
    <title>input</title>
    <meta name="viewport" content="width=device-width, initial-scale=1">
  </head>
  <body>
    <p>{ "input" : "./inputs" }</p>
  </body>
</html>
```

### Expected Behaviour
 When a valid `config` file is passed to the tool, it should create an HTML version of all files inside the `./inputs` directory.

**Review by SerpentBytes — APPROVED** [review]

**Comment by Ririio** [comment]
Problem 1: I fixed the issue regarding that
Problem 2: I added a success status message
Problem 3: That's not really for my config file, I tested your input and you don't go through multi-layer directories even with it 

**Running the command**
image

**HTML contents**
image

**Web Output**
image

**Clicking the link**
image

The .json config file should have 

`{
  "input": "./inputs/Sherlock-Holmes-Selected-Stories"
}`

for it specifies the directory and its contents

**Review by SerpentBytes — APPROVED** [review]

**Pull request merged**